recipes/libsass/all/conanfile.py

```diff
@@ -14,18 +16,24 @@ class LibsassConan(ConanFile):
     options = {"shared": [True, False], "fPIC": [True, False]}
     default_options = {"shared": False, "fPIC": True}
 
-    build_requires = "autoconf/2.69", "libtool/2.4.6"
-
     _autotools = None
 
     @property
     def _source_subfolder(self):
         return "source_subfolder"
 
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
     def validate(self):
```

Comment:
```suggestion
```
This should work around this bug: https://github.com/conan-io/conan-center-index/pull/4182#issuecomment-758889526

Reply:
yes, we see the advantage to keep early `ConanInvalidConfiguration` in `configure()`.

Is `sassc` compatible with Windows? Shouldn't you raise in `sassc` recipe also?

Reply:
I'll raise the exception in sassc too.
oth recipe can be made compatible with windows, it should be buildable with the provided visual studio solution, but I have no need for it and no time to work on this, so maybe later

Reply:
I'll try to support MinGW in libsass, so only Visual Studio will be unsupported for the moment.